pkg/shell/shell-modals.jsx

```diff
@@ -86,6 +86,8 @@ export const AboutCockpitModal = () => {
 
 export const LangModal = () => {
     const language = document.cookie.replace(/(?:(?:^|.*;\s*)CockpitLang\s*=\s*([^;]*).*$)|^.*$/, "$1") || "en-us";
+    const lang_dir = language.match(/he-il/) ? "rtl" : "ltr";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-14107-20221130-103219-f4612325-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
Given how brittle this determination is, this duplication with base_index.js needs to go.

This would actually feel better in cockpit.js, so that it will apply to the shell and all frames, and all third-party projects as well. This would also avoid the duplication.